.github/workflows/todo.yml

```diff
@@ -19,4 +20,25 @@ jobs:
       run: docker run --env-file .env.test travis_todo todo_app/test/test_appTest.py
     - name: Run end-to-end test
       # You'll need to adjust this to also pass in the relevant environment variables
-      run: docker run --env TOKEN --env KEY --env  SECRET_KEY travis_todo todo_app/test/test_appUiTest.py
\ No newline at end of file
+      run: docker run --env TOKEN --env KEY --env  SECRET_KEY travis_todo todo_app/test/test_appUiTest.py
+
+  deployingtoheroku:
+    name: heroku
+    runs-on: ubuntu-latest
+    # Only deploy when the test pass
+    needs: [test]
+    # if: contains('refs/heads/production', github.ref)  
+    env:
+      HEROKU_API_KEY: ${{secrets.HEROKU_API_KEY}}
+      TOKEN: ${{ secrets.trelloToken}}
+      KEY: ${{ secrets.trelloKey}}
+      SECRET_KEY: ${{ secrets.trelloSecretKey}}
```

Comment:
I don't think you need these three here, since they should already be configured in Heroku, you just need the Heroku API key.